_visual-editor/README.md

```diff
@@ -0,0 +1,381 @@
+# Migrating from `flutter-quill` to `visual-editor`
+
+> [!WARNING]
+>
+> At the time of writing,
+> `visual-editor` is still in development and **does not offer all the features `flutter-quill` does**.
+> There are a few bugs that are noticeable (more specifically, text selection) 
+> that do not work properly on mobile devices.
+>
+> We will use a **fork of `visual-editor`** - https://github.com/LuchoTurtle/visual-editor in this guide.
```

Comment:
Please don't `fork` repos to your own `GitHub`.
That's like making a copy of the key to the company building and keeping it in your own safe so that you have it when you leave the company. shady. 
If you're working on something for @dwyl do it in the @dwyl org so that other people can contribute seamlessly.

Reply:
Resolved in [`6634ccf` (#7)](https://github.com/dwyl/flutter-wysiwyg-editor-tutorial/pull/7/commits/6634ccf244922c70d94b6bf176b00a487eb086b3). Thanks 👌